lang/cs/Org.Apache.REEF.Client/API/IREEFClient.cs

```diff
@@ -43,5 +44,12 @@ public interface IREEFClient
         /// <returns>IDriverHttpEndpoint</returns>
         [Unstable("0.13", "Working in progress for what to return after submit")]
         IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission);
+
+        /// <summary>
+        /// Returns the Application status in running the job
+        /// </summary>
+        /// <returns></returns>
+        [Unstable("0.14", "Working in progress for rest API status returned")]
+        Task<ApplicationState> GetJobStatus();
```

Comment:
Job status needs to keep pulling after the Submit() is returned.. Therefore we need to separate API for it.